lib/Runtime/Base/CompactCounters.h

```diff
@@ -18,24 +18,23 @@ namespace Js
         bool isCleaningUp;
 #endif
         union {
-            uint8 u8Fields[CountT::Max];
-            int8 i8Fields[CountT::Max];
-            WriteBarrierPtr<uint16> u16Fields;
-            WriteBarrierPtr<uint32> u32Fields;
-            WriteBarrierPtr<int16> i16Fields;
-            WriteBarrierPtr<int32> i32Fields;
+            uint8 u8Fields[1];
```

Comment:
Can do this on 8 bit ones, still need plus size when expanding. It can make 8bit one debugging easier though, let me change it